src/pages/injector/app.js

```diff
@@ -91,6 +66,10 @@ export const App = () => {
     setSaveStatus('idle')
   }
 
+  const closePanel = () => {
+    setSaveStatus('idle')
+  }
+
```

Comment:
So we can pass this down and close the panel when you click enter in the tagging input.